tests/test_pool.py

```diff
@@ -527,3 +527,34 @@ def sleep(conn):
         yield from cur.execute('SELECT 1;')
         val = yield from cur.fetchone()
         assert (1,) == val
+
+
+@pytest.mark.run_loop
+def test_drop_connection_if_timedout(create_pool, loop):
+
+    @asyncio.coroutine
+    def _set_global_conn_timeout(pool, t):
+        # Set timeout for client connection
+        # https://www.postgresql.org/docs/current/static/runtime-config-client.html
+        conn = yield from pool.acquire()
+        cur = yield from conn.cursor()
+        yield from cur.execute('set statement_timeout to %s;' % t)
```

Comment:
I am not sure that this is correct, `statement_timeout` is timeout for statement execution time, not the timeout for idle connection.